src/i18n.ts

```diff
@@ -5,11 +5,7 @@ export const languages = [
   { lng: "de", label: "Deutsch", data: require("./locales/de.json") },
   { lng: "es", label: "Español", data: require("./locales/es.json") },
   { lng: "fr", label: "Français", data: require("./locales/fr.json") },
-  {
-    lng: "nb-no",
-    label: "Norsk Bokmål",
-    data: require("./locales/nb-no.json"),
-  },
+  { lng: "no", label: "Norsk", data: require("./locales/no.json") },
```

Comment:
Is this change correct?

Reply:
Yes, I did that manually since we have only one Norwegian language

https://github.com/excalidraw/excalidraw/blob/master/crowdin.yml#L3

Reply:
If needed we will add special cases later.. it wasn't that straight forward with automation.